lights/light_regulator/LampRegulator.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Sockets;
+using System.Text;
+
+namespace Lamp_Regulator 
+{
+    class Regulator
+    {
+        private byte[] _buffer = new byte[1024];
+        private List<Socket> _clientSockets = new List<Socket>();
+        private Socket _serverSocket = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
+        private int BACKLOG;
```

Comment:
to nie sa stałe, tylko podawane w argumencie konstruktora. Moim zdaniem napisane małymi literami będzie bardziej czytelne. `PORT` podobnie